xdrip-get-entries.sh

```diff
@@ -1371,4 +1375,22 @@ function check_sensitivity()
   sensitivity=$(jq .ratio ${HOME}/myopenaps/settings/autosens.json)
 }
 
+function bt_watchdog()
+{
+  logfiledir=/var/log/openaps
+  logfilename=logger-reset-log.txt
+  minutes=14
+  xdrip_errors=`find ${HOME}/myopenaps/monitor/xdripjs -mmin -$minutes -type f -name "*entry.json" | grep entry.json; find $logfiledir -mmin -$minutes -type f -name $logfilename`
+  if [ -z "$xdrip_errors" ]
+  then
+    logfile=$logfiledir/$logfilename
+    date >> $logfile
+    echo "no entry.json for $minutes minutes - rebooting" | tee -a $logfile
+    wall "Rebooting to fix BT and xdrip-js!"
+    cd ${HOME}/myopenaps && /etc/init.d/cron stop && killall -g openaps ; killall -g oref0-pump-loop | tee -a $logfile
+    sleep 5
```

Comment:
What if someone is editing a file? Should we give them 15 seconds to save and get out?

Reply:
ok, will update to 15 and make the message clearer.